build.py

```diff
@@ -96,8 +101,11 @@ def create_zip_file(source_dir, target_file):
     target_dir = os.path.dirname(target_file)
     if not os.path.exists(target_dir):
         os.makedirs(target_dir)
-    cd(source_dir)
-    run('zip', '-r', target_file, '.')
+    base, ext = os.path.splitext(target_file)
+    shutil.make_archive(
+        base if ext == '.zip' else target_file,
```

Comment:
This function is called `create_zip_file` and the rest of the code ensures it is always a zip file so it shouldn't have code to handle files that aren't zip files.